README.md

```diff
@@ -7,7 +7,15 @@ For a guide of how to use OneConfig please see our documentation at https://docs
 ![image](https://user-images.githubusercontent.com/67508414/172061081-7a32a139-d5c6-4f5a-907a-5af66deb799f.png)
 <br><br><br>
 
-### Supporting Projects 
+## License
+
+OneConfig free and open-source. Except where noted otherwise (below and/or in individual files), all code is licensed under BOTH 
```

Comment:
shouldn't this be
```suggestion
OneConfig is free and open-source. Except where noted otherwise (below and/or in individual files), all code is licensed under BOTH 
```

Reply:
this is a certified moontidez moment, ill fix it